target/linux/mediatek/patches-6.1/250-pinctrl-mediatek-mt7981-add-additional-uart-group.patch

```diff
@@ -0,0 +1,40 @@
+mediatek: mt7981: pinctrl: add additional uart group
```

Comment:
Has this been sent upstream?
If so, please add `X-Patchwork-*` headers (along with other RFC822 headers which would be needed to re-send the patch upstream)

Reply:
Yes, the patch has been merged upstream in linusw tree (https://git.kernel.org/pub/scm/linux/kernel/git/linusw/linux-pinctrl.git/commit/?h=for-next&id=8714b3414dd6070609fabdd73a70e5f818f77f24). I've taken the patch from https://patchwork.kernel.org in order to add the requested headers, please let me know if any are missing.